src/bin/avml.rs

```diff
@@ -29,6 +29,14 @@ struct Config {
     #[arg(long, value_enum)]
     source: Option<Source>,
 
+    /// Specify the maximum estimated disk usage (in MB)
+    #[arg(long)]
+    max_disk_usage: Option<NonZeroU64>,
+
+    /// Specify the minimum estimated free space (in percentage) to stay under
+    #[arg(long, value_parser = min_disk_usage_percentage)]
+    min_free_space_percentage: Option<f64>,
```

Comment:
Did you consider specifying this as a maximum percentage instead? Most production alert monitoring tools I've encountered tend to express alert conditions in terms of disk utilization rather than available space (i.e. "90% consumed" rather than "10% free").

Reply:
Yep, I think in practice, I implemented "stay under this percentage" but named it min-free-space-percentage.

I will adjust the variable & argument naming such that this is consistent with the idea of "stay under x percent".